### PR TITLE
Handle errors in test handler

### PR DESCRIPTION
**Pull request by evaninja**
**JIRA**: https://clever.atlassian.net/browse/IPOC-295

**Details**: Fixing a bug with the test command where we were ignoring errors previously.

**Testing**:
- [x] Run Flarebot test in `bots`

**Roll Out**:

### CODE REVIEW

**Comment by bstein-clever** [comment]
Nit: `fixing a big`

File: main.go [hunk 1]
```diff
@@ -237,29 +237,55 @@ func main() {
...
+		channel, err := client.api.GetChannelInfo(msg.Channel)
+		if err != nil {
+			client.Send("Unable to determine channel info", msg.Channel)
+			return
+		}
 
 		client.Send(fmt.Sprintf("this channel is %s", channel.Name), msg.Channel)
 
-		ticket, _ := JiraServer.GetTicketByKey("flare-165")
+		ticket, err := JiraServer.GetTicketByKey("flare-165")
```
Comment (bstein-clever) [inline main.go]: Why is this hard-coded to 165?

File: main.go [hunk 2]
```diff
@@ -237,29 +237,55 @@ func main() {
...
+		if err != nil {
+			client.Send("Unable to find JIRA ticket by key: flare-165", msg.Channel)
+			return
+		}
 
 		fmt.Println(ticket)
 
 		// get a test google doc and update it
 		googleDocID := "1Hd2T9hr4wYQZY6ZoZJG3y3yc6zuABjLumPQccHI1XXw"
```
Comment (bstein-clever) [inline main.go]: Ditto. Is this a test path? It's still main.go? :(

**Comment by bstein-clever** [comment]
A couple of questions, otherwise LGTM

File: main.go (hunk 1 above)
Comment (evaninja) [inline main.go]: I don't know. Good question

File: main.go (hunk 2 above)
Comment (evaninja) [inline main.go]: Yep, it's the path that runs when you type `@flarebot test`

File: main.go (hunk 1 above)
Comment (evaninja) [inline main.go]: Clarified in eed2d2a

File: main.go (hunk 2 above)
Comment (evaninja) [inline main.go]: Updated this to check for the validity of the template file instead in eed2d2a

**Pull request merged**